app/controllers/admin/petitions_controller.rb

```diff
@@ -33,24 +33,18 @@ def resend
     redirect_to admin_petition_url(@petition), notice: :email_resent_to_creator
   end
 
-  def copy_content
```

Comment:
is this not needed anymore?

Reply:
No, I added a content controller so it could be reset that models the content as a singleton resource: https://github.com/ScottishParliament/e-petitions/commit/257db9c9b3e7ee83686a839658475a4dc9cd9a56